client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -492,6 +492,21 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         MOBILE_APP_LOCALE("mobileAppLocale", "en_US", false),
 
         /**
+         * This parameter represents the appium device.
+         */
+        MOBILE_DEVICE("mobileDevice", "", false),
+
+        /**
+         * This parameter represents the appium device platform version.
+         */
+        MOBILE_PLATFORM_VERSION("mobilePlatformVersion", "", false),
```

Comment:
I'm asking you to combine them into a single ConfigProperty for consistency with the annotation. Please re-read my original comment. The use case  you mention would still be supportable.